lib/polymorphic_integer_type/extensions.rb

```diff
@@ -59,6 +83,22 @@ def remove_type_and_establish_mapping(name, options, scope)
         end
       end
 
+      def retrieve_polymorphic_type_mapping(polymorphic_type:, class_name:)
+        return if polymorphic_type == nil
```

Comment:
`.nil?`?

Reply:
😄  The Sandi Metz training instilled the idea of equality checks over all other conditionals. For this case, I don't think entirely necessary. I'll use `nil?` instead.